addons/website/static/src/scss/website.scss

```diff
@@ -364,6 +364,13 @@ table.table_desc tr td {
 // Snippets
 //
 
+// Cover
+.s_cover {
+    min-height: 100vh;
+    display: flex;
+    align-items: center;
+}
```

Comment:
- I think that the vertical centering is part of the alignment option on rows, no need of css for that.
- Forcing a min-height has always been a bad idea and we try to avoid those for the last 5 years... maybe it should be an option, to check the usability with SBU maybe

Reply:
sbu changed his mind, reverting this